python/lsst/ts/ofc/state_estimator.py

```diff
@@ -23,8 +23,13 @@
 
 import logging
 
+import galsim
 import numpy as np
 
+from . import SensitivityMatrix
+from .ofc_data import OFCData
+from .utils.intrinsic_zernikes import get_intrinsic_zernikes
+
```

Comment:
Is this ordered by `isort`?

Reply:
I have run isort and it doesn't complain or make any changes. Is there anything in particular that worries you?

Reply:
Just felt the order is a little weird. But if `isort` does not complain, it should be fine. Thanks to double check this!